.github/workflows/test-gpu.yml

```diff
@@ -0,0 +1,43 @@
+name: Test GPU
+
+on: [pull_request]
+
+jobs:
+  build-test:
+    runs-on: [self-hosted, gpu]
+    strategy:
+      matrix:
+        include:
+          - itk-git-tag: "c693dbc830859ea10d573301f075e121a4c6c581"
+            cmake-build-type: "MinSizeRel"
+        
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Doesn't this only check out the current repo, `ITKVkFFTBackend`?

Reply:
Yes, workflow has been adjusted to get ITK and OpenCL-ICD-Loader as well.